build.sbt

```diff
@@ -56,13 +56,13 @@ libraryDependencies ++= Seq(
     "com.gu" %% "identity-test-users" % "0.6",
     "com.gu" %% "content-authorisation-common" % "0.4",
     "com.gu" %% "tip" % "0.1.1",
-    "com.gu" %% "acquisition-event-producer-play26" % "4.0.26",
     "com.github.nscala-time" %% "nscala-time" % "2.16.0",
     "io.sentry" % "sentry-logback" % "1.7.5",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2",
     "org.scalatest" %% "scalatest" % scalatestVersion % "test",
     "org.scalactic" %% "scalactic" % scalatestVersion % "test",
-    "org.seleniumhq.selenium" % "selenium-java" % "3.7.1" % "test",
+    "org.scalatestplus" %% "selenium-3-141" % (scalatestVersion + ".0") % "test",
```

Comment:
the 3.141 has to match the `selenium-java` version, and the actual version needs to match scalatest, with a point release appended (which happens to be `0`)